src/ChromeView.js

```diff
@@ -52,34 +61,20 @@ export default class Chrome {
       .attr('fill', tinycolor(this._props.bgColor)
         .isLight() ? '#000' : '#FFF');
 
-    this._props.$context.append(this._$view);
+    this.showExtras();
+
+    this._$context.append(this._$view);
   }
 
   renderSlide(slide) {
-    const picture = slide.$node.data('picture');
-    const src = slide.$node.data(this._props.imageSrcDataAttr);
-    let $content;
-
-    if (picture) {
-      $content = $('<picture />');
-      picture.sources.forEach(({ srcset, media_query: media }) => {
-        $('<source />', { srcset, media }).appendTo($content);
-      });
-      const { src, alt } = picture.img;
-      $('<img />', { src, alt }).appendTo($content);
-    }
-    else {
-      $content = $('<img />', { src: src });
-    }
-
-    this._$contents.empty().append($content);
+    this._$contents.empty().append(this._getSlideContent(slide));
```

Comment:
In fact, `.html()` is just shorthand for `.empty().append()`.

https://github.com/jquery/jquery/blob/22449eb/src/manipulation.js#L425